tests/testthat/test-variable-catalog.R

```diff
@@ -40,22 +39,22 @@ with_mock_crunch({
         expect_length(varcat, 6)
         expect_identical(active(hidden(varcat)), hidden(active(varcat)))
     })
-
+    
     gender.url <- "https://app.crunch.io/api/datasets/1/variables/gender/"
     test_that("Extract methods: character and numeric", {
         expect_is(varcat[[gender.url]], "VariableTuple")
         expect_identical(varcat[[gender.url]]@body,
             index(varcat)[[gender.url]])
         expect_identical(index(varcat[2:3]), index(varcat)[2:3])
     })
-
+    
```

Comment:
I'm not sure why it's not default, but I've tried to keep with linted style using the Strip trailing horizontal whitespace when saving setting (a good argument for adding .Rproj to the repo like you suggested!)